### DISCUSSION TITLE
Bright Green Background

### DISCUSSION THREAD

**Discussion by seehad** [NvChad/NvChad#3130] [Q&A] [answered]
I run the install clone and my nvim is bright green and virtually unreadable.  All I've done is

brew install nvim
git clone https://github.com/NvChad/starter ~/.config/nvim && nvim
<img width="748" alt="Screenshot 2024-12-16 at 6 39 39 PM" src="https://github.com/user-attachments/assets/1cc4b62a-7d1d-4000-ac6b-4f2462f8470c" />

What's wrong?

**Comment by siduck** [comment, marked as answer]
enable true colors in your terminal
